bbc/simorgh
# MediaLoader on Live radio page

Resolves JIRA https://jira.dev.bbc.co.uk/browse/WSTEAM1-1288

Overall changes
======
- Removes iframe media player from Live Radio page in favour of new MediaLoader
- Adds new `liveRadio` config to MediaLoader to create SMP config for Live Radio streams
- Uplifts LiveRadioPage to TS

Testing
======
1. Visit http://localhost.bbc.com:7080/hausa/bbc_hausa_radio/liveradio?renderer_env=live
2. Confirm media player renders and allows playback, showing all related controls for Live content

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
